projects/m4/011-distinct-names/js/index.js

```diff
@@ -0,0 +1,62 @@
+const fs = require("fs");
+const fsPromises = fs.promises;
+const path = require("path");
+const data = path.join(__dirname, "data");
+
+// MAIN FUNCTION
+// Read all the files from a directory and print all the unique popular names (female and male)
+fs.readdir(data, (err, files) => {
```

Comment:
I think it's better to give the function a more meaningful name. 
Example: `getUniqueNames`